packages/angular_devkit/build_angular/src/babel/presets/application.ts

```diff
@@ -98,11 +101,47 @@ function createI18nPlugins(
   return plugins;
 }
 
+function createNgtscLogger(
+  reporter: DiagnosticReporter | undefined,
+): import('@angular/compiler-cli/src/ngtsc/logging').Logger {
+  return {
+    level: 1, // Info level
```

Comment:
Nit: since we don't use the `info` messages shouldn't this be changed to `warning`.
https://github.com/angular/angular-cli/pull/19641/files#diff-c36474716c699264f5af41ac0f4fb12c6e0a8580e9063813b20fff48e5f43548R107